### DISCUSSION TITLE
Bug in running

### DISCUSSION THREAD

**Discussion by hereagain-Y** [PGScatalog/pgsc_calc#269] [Q&A]
When I tried to run command  nextflow run pgscatalog/pgsc_calc -profile test, conda/singularity -r  v2.0.0-alpha.4, result on  conda and signularity running result both shows ERROR ~ Error executing process > 'PGSCATALOG_PGSCALC:PGSCALC:INPUT_CHECK:SAMPLESHEET_JSON (samplesheet.csv)'

Caused by:
  Process `PGSCATALOG_PGSCALC:PGSCALC:INPUT_CHECK:SAMPLESHEET_JSON (samplesheet.csv)` terminated with an error exit status (127), I don't know where this comes from. I've also update the pgscatalog-utils pkg to the latest version.

**Comment by nebfield** [comment]
Could you test with the [latest release please](https://github.com/PGScatalog/pgsc_calc/releases/tag/v2.0.0-alpha.5):

```
$  nextflow run pgscatalog/pgsc_calc -profile test,singularity -latest
```

or

```
$  nextflow run pgscatalog/pgsc_calc -profile test,conda -latest
```

Exit status 127 means a command/program isn't found on your computer, but the profiles should automatically install all the software you need. You shouldn't need to install or update `pgscatalog_utils` manually with `pip`. Make sure not to include any whitespace in the comma separated list of profiles, e.g.:

```
-profile test,singularity ✅ 
-profile test, singularity ❌ 
```

**Reply by hereagain-Y** [reply]
Yes, that works for me. Thank you for your intime response :)
